models/tridimensional/data_kleinstiver/Analysis of PyRosetta Output.py

```diff
@@ -0,0 +1,72 @@
+
+import scipy, scipy.stats.kendalltau, scipy.stats.chisquare
+
+def los_clnr(los):
+    """
+    Consumes a list of strings containing newline characters and commas, and 
+    cleans the strings by removing the newline characters and splitting by 
+    commas before returning a list of lists, where each list corresponds to
+    the important data from each string
+    """
+    newline_remove = map(lambda (el): el.strip(), los)
+    comma_split = map(lambda (el): el.split(','), newline_remove)
+    as_ints = [comma_split[0]] + map(lambda (el): map(float, el), comma_split[1:])
+    return as_ints
+
+def sort_data(lst):
+    """
+    Consumes a list produced by reading a csv file with titled columns
+    corresponding to PyRosetta outputs
+    """
+    cleaned_data = los_clnr(lst)
+    titles = cleaned_data[0]
+    just_vals = cleaned_data[1:]
+    properly_sorted_lst = []
+    for i in range(len(titles)):
+        i_lst = []
+        for val_lst in just_vals:
+            i_lst.append(val_lst[i])
+        properly_sorted_lst.append(i_lst)
```

Comment:
I think this replaces lines 24-29 but should be tested:
`data_transposed = [[just_vals[row][col] for row in xrange(len(just_vals[0]))] for col in xrange(len(titles))]`

If it does its also faster (append is really slow in python). xrange also faster than range when actual lists aren't needed.